docs/data/material/customization/css-theme-variables/usage.md

```diff
@@ -0,0 +1,394 @@
+# CSS theme variables - Usage
+
+<p class="description">Learn how to adopt CSS theme variables.</p>
+
+## Getting started
+
+The CSS variables API relies on a provider called `CssVarsProvider` to inject styles into Material UI components.
+`CssVarsProvider` generates CSS variables out of all tokens in the theme that are serializable, and makes them available in the React context along with the theme itself via [`ThemeProvider`](/material-ui/customization/theming/#theme-provider).
+
+Once the `App` renders on the screen, you will see the CSS theme variables in the HTML `:root` stylesheet.
+The variables are flattened and prefixed with `--mui` by default:
+
+<codeblock>
+
+```jsx JSX
+import { CssVarsProvider } from '@mui/material/styles';
+
+function App() {
+  return <CssVarsProvider>{/* ...you app */}</CssVarsProvider>;
+}
+```
+
+```css CSS
+:root {
+  --mui-palette-primary-main: #1976d2;
+  --mui-palette-primary-light: #42a5f5;
+  --mui-palette-primary-dark: #1565c0;
+  --mui-palette-primary-contrastText: #fff;
+  /* ...other variables */
+}
+```
+
+</codeblock>
+
+:::info
+The `CssVarsProvider` is built on top of the [`ThemeProvider`](/material-ui/customization/theming/#themeprovider) with extra features like CSS variable generation, storage synchronization, unlimited color schemes, and more.
+
+If you have an existing theme, you can migrate to CSS theme variables by following the [migration guide](/material-ui/migration/migration-css-theme-variables/).
+:::
+
+## Dark mode only application
+
+To switch the default light to dark palette, set `colorSchemes: { dark: true }` to the `extendTheme`.
+Material UI will generate the dark palette instead.
+
+```jsx
+import { CssVarsProvider, extendTheme } from '@mui/material/styles';
+
+const theme = extendTheme({
+  colorSchemes: { dark: true },
+});
+
+function App() {
+  return <CssVarsProvider theme={theme}>{/* ...you app */}</CssVarsProvider>;
+}
+```
+
+## Light and dark mode application
+
+To support both light and dark modes, set `colorSchemes: { light: true, dark: true }` to the `extendTheme`.
+Material UI will generate both light and dark palette with [`@media (prefers-color-scheme)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).
+
+<codeblock>
+
+```jsx JSX
+import { CssVarsProvider, extendTheme } from '@mui/material/styles';
+
+const theme = extendTheme({
+  colorSchemeSelector: 'media',
```

Comment:
If I understand correctly, you either provide

`colorSchemes: { light: true, dark: true }`

or 

`colorSchemeSelector: 'media'`

right?

If this is the case, then I would show `colorSchemes: { light: true, dark: true }` for this example and leave `colorSchemeSelector` for the advanced section. I think `colorSchemes: { light: true, dark: true }` is more beginner friendly. What do you think?

Reply:
Opps, this is my mistake.

`colorSchemeSelector: 'media'` alone won't work because the there is only `light` color scheme by default, so `colorSchemes: { light: true, dark: true }` is required.

I will update the docs.

Reply:
Done!